YoCode/HelperMethods.cs

```diff
@@ -65,5 +67,41 @@ public static bool ListContainsAnyKeywords(this List<string> list, IEnumerable<s
             }
             return false;
         }
+
+        public static List<string> GetMultipleLinesWithAllKeywords(this string text, List<string> keywords)
+        {
+            var list = new List<string>();
+            var sr = new StringReader(text);
+            string line;
+            while ((line = sr.ReadLine()) != null){
+                if(line.ContainsAll(keywords))
+                {
+                    list.Add(line);
+                }
+
+            }
+            return list;
+        }
+    
+        public static string GetStringBetweenStrings(this string line, string fromString,string toString)
+        {
+            int pFrom = line.IndexOf(fromString) + fromString.Length;
+            int pTo = line.LastIndexOf(toString);
+
+            return line.Substring(pFrom, pTo - pFrom);
+        }
+
+        public static bool SubjectivelyEquals(this double a, double b)
+        {
+            double aComp = Math.Round(a, RoundTo);
+            double bComp = Math.Round(b, RoundTo);
+
+            if(Math.Abs(aComp - bComp) > 0.000001)
```

Comment:
This `if` statement could be replaced with `return (Math.Abs(aComp - bComp) <= 0.000001);`

Reply:
Technically, I think the 0.000001 should actually be a value that scales with the number (in other words, it should be a relative value). But since we're never going to be testing with numbers that get really big, it will make no difference in this case.